new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -0,0 +1,780 @@
+            .value(WDT_VERSION)) // WDT version to use
+        .addEnvItem(new V1EnvVar()
+            .name("WDT_MODEL_FILE")
+            .value("/u01/weblogic/" + modelFile.getFileName())) // WDT model file
+        .addEnvItem(new V1EnvVar()
+            .name("WDT_VAR_FILE")
+            .value("/u01/weblogic/" + domainPropertiesFile.getFileName())) // WDT model property file
+        .addEnvItem(new V1EnvVar()
+            .name("DOMAIN_HOME_DIR")
+            .value("/shared/domains/" + domainUid)); // domain location
+
+    logger.info("Running a Kubernetes job to create the domain");
+    runCreateDomainJob(pvName, pvcName, domainScriptConfigMapName, namespace, jobCreationContainer);
+
+  }
+
+  /**
+   * Create configmap containing domain creation scripts.
+   *
+   * @param configMapName name of the configmap to create
+   * @param files files to add in configmap
+   * @param namespace name of the namespace in which to create configmap
+   * @throws IOException when reading the domain script files fail
+   * @throws ApiException if create configmap fails
+   */
+  private void createConfigMapForDomainCreation(String configMapName, List<Path> files, String namespace)
+      throws ApiException, IOException {
+    logger.info("Creating configmap {0}", configMapName);
+
+    Path domainScriptsDir = Files.createDirectories(
+          Paths.get(TestConstants.LOGS_DIR, this.getClass().getSimpleName(), namespace));
+
+    // add domain creation scripts and properties files to the configmap
+    Map<String, String> data = new HashMap<>();
+    for (Path file : files) {
+      logger.info("Adding file {0} in configmap", file);
+      data.put(file.getFileName().toString(), Files.readString(file));
+      logger.info("Making a copy of file {0} to {1} for diagnostic purposes", file,
+          domainScriptsDir.resolve(file.getFileName()));
+      Files.copy(file, domainScriptsDir.resolve(file.getFileName()));
+    }
+    V1ObjectMeta meta = new V1ObjectMeta()
+        .name(configMapName)
+        .namespace(namespace);
+    V1ConfigMap configMap = new V1ConfigMap()
+        .data(data)
+        .metadata(meta);
+
+    boolean cmCreated = assertDoesNotThrow(() -> createConfigMap(configMap),
+        String.format("Failed to create configmap %s with files %s", configMapName, files));
+    assertTrue(cmCreated, String.format("Failed while creating ConfigMap %s", configMapName));
+  }
+
+  /**
+   * Create a job to create a domain in persistent volume.
+   *
+   * @param pvName name of the persistent volume to create domain in
+   * @param pvcName name of the persistent volume claim
+   * @param domainScriptCM configmap holding domain creation script files
+   * @param namespace name of the domain namespace in which the job is created
+   * @param jobContainer V1Container with job commands to create domain
+   * @throws ApiException when Kubernetes cluster query fails
+   */
+  private void runCreateDomainJob(String pvName,
```

Comment:
may be method name should be modified to createDomainJob()

Reply:
Fixed